src/schema/paddle.ts

```diff
@@ -0,0 +1,100 @@
+import type { CountryCode } from '@paddle/paddle-node-sdk';
+import type { BaseContext } from '../Context';
+import { paddleInstance } from '../common/paddle';
+import { remoteConfig } from '../remoteConfig';
+import type { IResolvers } from '@graphql-tools/utils';
+import { traceResolvers } from './trace';
+
+export const typeDefs = /* GraphQL */ `
+  """
+  Price details for a product
+  """
+  type Price {
+    """
+    Label of the price
+    """
+    label: String!
+    """
+    Id of the price
+    """
+    value: String!
+    """
+    Formatted price with currency symbol
+    """
+    price: String!
+    """
+    Raw unformatted price value
+    """
+    priceUnformatted: String!
+    """
+    Three letter currency code (e.g. USD, EUR)
+    """
+    currencyCode: String!
+    """
+    Optional additional label text
+    """
+    extraLabel: String
+    """
+    Apps id
+    """
+    appsId: String
+  }
+
+  """
+  Price previews
+  """
+  type PricePreviews {
+    """
+    Three letter currency code
+    """
+    currencyCode: String!
+    """
+    Price previews
+    """
+    items: [Price!]!
+  }
+
+  extend type Query {
+    pricePreviews: PricePreviews!
+  }
+`;
+
+export interface GQLCustomData {
+  appsId: string;
+  label: string;
+}
+
+export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
+  unknown,
+  BaseContext
+>({
+  Query: {
+    pricePreviews: async (_, __, ctx: BaseContext) => {
+      const region = ctx.req.headers['x-client-region'] as
+        | CountryCode
+        | undefined;
+      const pricePreview = await paddleInstance?.pricingPreview.preview({
+        items: Object.keys(remoteConfig.vars.pricingIds!).map((priceId) => ({
```

Comment:
```suggestion
        items: Object.keys(remoteConfig.vars.pricingIds || {}).map((priceId) => ({
```

All remote config keys must be treated as optional, we should fallback to `{}` or throw error if pricing ids is not set.